querybook/webapp/ui/Comment/Comment.tsx

```diff
@@ -17,19 +18,21 @@ import { Reactions } from './Reactions';
 interface IProps {
     comment: IComment;
     editComment: (text: DraftJS.ContentState) => void;
+    deleteComment: () => void;
     isBeingEdited: boolean;
     isBeingRepliedTo: boolean;
     isChild: boolean;
-    createChildComment: () => void;
+    editChildComment: () => void;
```

Comment:
onEditChildComment or startEditChildComment?

otherwise the function sounds like a direct modification which is strange because it has no parameters